packages/allure-hermione/test/.hermione.conf.js

```diff
@@ -9,8 +9,8 @@ module.exports = {
   browsers: {
     headless: {
       desiredCapabilities: {
-        browserName: "chrome",
-        "goog:chromeOptions": {
+        browserName: "firefox",
```

Comment:
Did that due `chromedriver` instability. `geckodriver` works much more stable and doesn't have conflicts with locally installed browser and selenium grid one.